README.md

```diff
@@ -1,16 +1,68 @@
 # helm-migrate-values
 
-## Documentation
-
-- [Install](#install)
-
+A plugin to migrate user-specified Helm values between chart versions when the schema of `values.yaml` changes. Define migration paths with migration files in the chart repo to ensure seamless upgrades.
 ## Requirements
 
-- Helm version 3 or newer
+- **Helm** version 3 or newer
+- **Git** as the `helm plugin install` command requires it
```

Comment:
Given that you can install this from a http server and by using the plugins folder, this should probably be reworded to something along these lines:
```suggestion
- **Git** if installing using the `helm plugin install` command below
```